lib/matplotlib/backends/backend_ps.py

```diff
@@ -286,9 +286,17 @@ def draw_image(self, gc, x, y, im, transform=None):
         h, w = im.shape[:2]
         imagecmd = "false 3 colorimage"
         data = im[::-1, :, :3]  # Vertically flipped rgb values.
-        # data.tobytes().hex() has no spaces, so can be linewrapped by relying
-        # on textwrap.fill breaking long words.
-        hexlines = textwrap.fill(data.tobytes().hex(), 128)
+        # data.tobytes().hex() has no spaces, so can be linewrapped by simply
+        # splitting data every nchars. It's equivalent to textwrap.fill only
+        # 3 orders of magnitude faster. See matplotlib#17176
```

Comment:
I "only" get a 100-fold speedup; I think you can just write "-- much faster than textwrap.fill".  Also I don't think the issue needs to be linked here.

Reply:
applied in 0f889dc